src/TryMudBlazor.Client/wwwroot/editor/main.js

```diff
@@ -213,7 +213,7 @@ window.Try.CodeExecution = window.Try.CodeExecution || (function () {
             const cache = await caches.open('dotnet-resources-/');
 
             const cacheKeys = await cache.keys();
-            const userComponentsDllCacheKey = cacheKeys.find(x => x.url.indexOf('Try.UserComponents.dll') > -1);
+            const userComponentsDllCacheKey = cacheKeys.find(x => /Try\.UserComponents\.[^/]*\.dll/.test(x.url));
```

Comment:
They changed how it works, now the `cacheKeys` contains url as
```
https://localhost:5001/_framework/Try.UserComponents.5js11t3n4f.dll.sha256-sWuk/v/P19Mdwxa9TJKy8H5RCinjfd/iOybI4HtjLRk=
```
I had to change logic, we cannot use `x.url.indexOf('Try.UserComponents')` (without `.dll` prefix) because then it can land on the `.pdb` file.
This part could lead when click first time on "run" doesn't show any compiled html or throwing the alert message.

Reply:
Alternatively we can just disable the fingerprints:
```
<WasmFingerprintAssets>false</WasmFingerprintAssets>
```
and leave the code as it was before.